cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -420,6 +435,48 @@ handleSimpleNode scp runP p2pMode trace nodeTracers nc onKernel = do
     forM_ basicInfoItems $ \(LogObject nm mt content) ->
       traceNamedObject (appendName nm tr) (mt, content)
 
+#ifdef UNIX
+  updateVars :: LOMeta
```

Comment:
A comment and perhaps a better name for this function would be good. `updateVars` is very generic what about `nodeTopologyUpdate` or something similar?

Reply:
I went for `updateTopologyConfiguration`